composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import models
+import airflow.utils.db
+import internal_unit_testing
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="module")
+def initalize_airflow_database():
+    airflow.utils.db.initdb()
+
+
+def test_dag_import():
+    models.Variable.set('gcp_project', 'python-docs-samples-tests')
```

Comment:
Can you get the project id from `GOOGLE_CLOUD_PROJECT` instead of the magic string?

Reply:
good catch - I realized that I should actually just be using a filler string here - we don't actually need to have our test project and there's no need to grab an environment variable. It's just a dummy variable that needs to exist - it doesn't matter what it's set to.